### DISCUSSION TITLE
How to run a cron job on both production and dev environment

### DISCUSSION THREAD

**Discussion by TeoChi-CF** [vercel/community#6701] [Help]
### Summary

I have to run a cron job on both production and dev environment which are part of the same project, how can I do that since crons only run on production?
Is the only way to recreate dev as a separate project?

### Steps to Reproduce

--

**Comment by versecafe** [comment]
@TeoChi-CF you can have your CRON point at an API on prod which then makes calls to your relevant dev branches, or if you want to hit all of your recent deployments you can use the vercel API to check for the x most recent deployments and get their URLs
